packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -5910,8 +5914,15 @@ export default class Meeting extends StatelessWebexPlugin {
    * @public
    * @memberof Meeting
    */
-  public leave(options: {resourceId?: string; reason?: any} = {} as any) {
+  public leave(
+    options: {
+      resourceId?: string;
+      reason?: any;
+      clientEventLeaveReason?: ClientEventLeaveReason;
```

Comment:
how are client apps supposed to know the difference between `reason` and `clientEventLeaveReason` and which one they should set? Do we really need 2 of them? Also jsdoc needs updating

Reply:
Updated js doc